cmd/skipper/main.go

```diff
@@ -172,6 +174,7 @@ func init() {
 	flag.StringVar(&applicationLogPrefix, "application-log-prefix", defaultApplicationLogPrefix, applicationLogPrefixUsage)
 	flag.StringVar(&accessLog, "access-log", "", accessLogUsage)
 	flag.BoolVar(&accessLogDisabled, "access-log-disabled", false, accessLogDisabledUsage)
+	flag.BoolVar(&accessLogJSONDisabled, "access-log-json-disabled", true, accessLogJSONDisabledUsage)
```

Comment:
Thanks for letting me know.
Used _accessLogJSONDisabled_ with default value as **true** just to follow same structure as seen in _accessLogDisabled_ where its default value is **false**.